lib/vtls/sectransp.c

```diff
@@ -2424,7 +2426,7 @@ static CURLcode sectransp_shutdown(struct Curl_cfilter *cf,
   struct st_ssl_backend_data *backend =
     (struct st_ssl_backend_data *)connssl->backend;
   CURLcode result = CURLE_OK;
-  ssize_t nread;
+  ssize_t nread = 0;
```

Comment:
Was this change intented? Is it silencing a warning?

Reply:
Yes. I added curl update to my projects and there is a warning about using nread uninitialized.
I thought I could add that here.

Reply:
Would you mind copying the warning here and your compiler/version? (It's kind of odd, looking at the code, but I may be misreading it)

Reply:
```
curl/lib/vtls/sectransp.c:2478:6: warning: variable 'nread' may be uninitialized when used here [-Wconditional-uninitialized]
 2478 |   if(nread > 0) {
      |      ^~~~~
curl/lib/vtls/sectransp.c:2429:16: note: initialize the variable 'nread' to silence this warning
 2429 |   ssize_t nread;
      |                ^
      |                 = 0

```

if you don't think this is important enough, then you can remove this change. But I compile with a ton of checks here and I would just add the 0 assignment and move on.

Reply:
Thanks! No probs, it's legit (I'm wondering why CI did not catch it!).

I think we're good to go.